tensorflow/java/src/gen/resources/tftypes.csv

```diff
@@ -0,0 +1,21 @@
+TF type,datatype code,Java type,Java boxed type,Creator?,default,Description

```

Comment:
I generally prefer to keep code and data resources separate. I think it *is* less readable if combined with the script, and we lose the functionality of viewing and editing it as a spreadsheet if they are combined.